src/app/core/components/manage-tags/manage-tags.component.ts

```diff
@@ -40,9 +42,13 @@ export class ManageTagsComponent implements OnInit {
   }
 
   public async deleteTag(tag: TagVO): Promise<void> {
-    this.tags = this.tags.filter((t) => t.tagId !== tag.tagId);
-    await this.api.tag.delete(tag);
-    this.refreshTags.emit();
+    this.prompt.confirm('Delete', 'Are you sure you want to delete this tag from all items in the current archive?').then(async () => {
+      this.tags = this.tags.filter((t) => t.tagId !== tag.tagId);
+      await this.api.tag.delete(tag);
+      this.refreshTags.emit();
+    }).catch(() => {
+      // do nothing
```

Comment:
If tag deletion fails, we should put the deleted tag back into the list. And probably log an error to Sentry. And maybe display an error message.